be/test/exec/CMakeLists.txt

```diff
@@ -35,11 +35,13 @@ ADD_BE_TEST(partitioned_hash_table_test)
 #ADD_BE_TEST(csv_scanner_test)
 #ADD_BE_TEST(csv_scan_node_test)
 # ADD_BE_TEST(csv_scan_bench_test)
+ADD_BE_TEST(parquet_scanner_test)
 ADD_BE_TEST(plain_text_line_reader_uncompressed_test)
 ADD_BE_TEST(plain_text_line_reader_gzip_test)
 ADD_BE_TEST(plain_text_line_reader_bzip_test)
 ADD_BE_TEST(plain_text_line_reader_lz4frame_test)
 ADD_BE_TEST(plain_text_line_reader_lzop_test)
+#ADD_BE_TEST(broker_parquet_reader_test)
```

Comment:
why you not compile this UT

Reply:
The ut depend on local real environment. for exmaple:    
 std::string path = "hdfs://A.B.C.D:8020/user/hive/warehouse/test.db/parquet_test/000000_0";